src/script/auth/page/Login.tsx

```diff
@@ -392,7 +398,7 @@ const mapStateToProps = (state: RootState) => ({
   isFetching: AuthSelector.isFetching(state),
   isSendingTwoFactorCode: AuthSelector.isSendingTwoFactorCode(state),
   loginData: AuthSelector.getLoginData(state),
-  loginError: AuthSelector.getError(state),
+  authError: AuthSelector.getError(state),
```

Comment:
This is not `loginError` it's more generic authError that can occur in some different places:

<img width="381" alt="Screenshot 2023-01-03 at 09 55 28" src="https://user-images.githubusercontent.com/45733298/210325764-e7d6ad0f-5a9d-42b9-8ab6-33b82eab681b.png">